DQMOffline/JetMET/src/JetAnalyzer_HeavyIons.cc

```diff
@@ -725,7 +567,10 @@ void JetAnalyzer_HeavyIons::analyze(const edm::Event& mEvent, const edm::EventSe
     Float_t SumSquaredCaloVsPt[etaBins_];
     Float_t SumSquaredCaloPt[etaBins_];
 
-    for(int i = 0;i<etaBins_;i++){
+    // Need to set up histograms to get the RMS values for each pT bin 
+    TH1F *hSumCaloVsPtInitial[nedge_pseudorapidity-1], *hSumCaloVsPt[nedge_pseudorapidity-1], *hSumCaloPt[nedge_pseudorapidity-1];
```

Comment:
Hi David, 
I do delete them in L721 onwards yeah? so im not sure what the issue is here? 
Cheers
Raghav